changes-set.txt

```diff
@@ -20,7 +20,6 @@ proposed  sylbid    biimtrd     etc.
 proposed  sylbird   biimtrrd    etc.
 proposed  syl5*     *trid       (syl5bi -> biimtrid; syl5eqel -> eqeltrid;etc.)
 proposed  syl6*     *trdi
-proposed  *mpt2*    *mpo*       (renaming process underway)
 (Please send any comments on these proposals to the mailing list or
 make a github issue.)
 
```

Comment:
I am unsure about cnmpt2res.  Can you confirm this need not be renamed?

Reply:
cnmpt2res is parallel to https://us.metamath.org/mpeuni/cnmpt1res.html and uses the same naming convention as seen in theorems such as https://us.metamath.org/mpeuni/cnmptk2.html (and others in this section). We elected not to change these at #3482